lib/livebook/intellisense/completion.ex

```diff
@@ -1,41 +1,48 @@
-defmodule Livebook.Completion do
+defmodule Livebook.Intellisense.Completion do
```

Comment:
Previously we would build the editor-like completion objects here. Now we return a list of detailed items, and then format the items either into completion items or detailed docs for hover.